smithy-aws-protocol-tests/model/restXml/http-string-payload.smithy

```diff
@@ -0,0 +1,70 @@
+$version: "2.0"
+
+namespace aws.protocoltests.restXml
+
+use smithy.test#httpRequestTests
+use smithy.test#httpResponseTests
+
+@http(uri: "/EnumPayload", method: "POST")
+@httpRequestTests([
+    {
+        id: "RestXmlEnumPayloadRequest",
+        uri: "/EnumPayload",
+        body: "enumvalue",
+        params: { payload: "enumvalue" },
+        method: "POST",
+        protocol: "aws.protocols#restXml"
+    }
+])
+@httpResponseTests([
+    {
+        id: "RestXmlEnumPayloadResponse",
+        body: "enumvalue",
+        params: { payload: "enumvalue" },
+        protocol: "aws.protocols#restXml",
+        code: 200
+    }
+])
+operation HttpEnumPayload {
```

Comment:
This operation and the one below need to be added to the [RestXml service](https://github.com/smithy-lang/smithy/blob/main/smithy-aws-protocol-tests/model/restXml/main.smithy#L13), otherwise code generators won't pick it up and generate the appropriate tests from it. See where the equivalent operations are added to the RestJson service [here](https://github.com/smithy-lang/smithy/blob/main/smithy-aws-protocol-tests/model/restJson1/main.smithy#L59-L60).

Reply:
added operations and fixed namespace